src/Equinor.ProCoSys.IPO.Query/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQueryHandler.cs

```diff
@@ -70,19 +123,19 @@ public class GetOutstandingIposForCurrentPersonQueryHandler : IRequestHandler<Ge
                 }
 
                 var outstandingIposResultDto = new OutstandingIposResultDto(
-                    currentUsersOutstandingInvitations.Select(invitation =>
-                    {
-                        var organization = invitation.Participants.First(p =>
-                            p.SignedBy == null &&
-                            (p.AzureOid == currentUserOid ||
-                            currentUsersFunctionalRoleCodes.Contains(p.FunctionalRoleCode))).Organization;
-                        return new OutstandingIpoDetailsDto
-                        {
-                            InvitationId = invitation.Id,
-                            Description = invitation.Description,
-                            Organization = organization
-                        };
-                    }));
+                       currentUsersOutstandingInvitations.Select(invitation =>
+                       {
+                           var organization = invitation.Participants.First(p =>
+                               p.SignedBy == null &&
+                               (p.AzureOid == currentUserOid ||
+                               currentUsersFunctionalRoleCodes.Contains(p.FunctionalRoleCode))).Organization;
+                           return new OutstandingIpoDetailsDto
+                           {
+                               InvitationId = invitation.Id,
+                               Description = invitation.Description,
+                               Organization = organization
+                           };
+                       }));
```

Comment:
Below there is a catch Exception which swallow the exception silently
1) Log the exception
2) Create a TECH-PBI, where we alter the logic to still log the exception **and** return 500-error. The client (Main) must handle this error and show a error-message like "something went wrong when getting outstanding ipos". 
Write in TECH-PBI: Remember to check if any else ipo endpoint used by Main catch Exceptions which swallow the exception silently

Reply:
Agree @bratli ?

Reply:
Yes

Reply:
TECH PBI Created; 96013